src/uct/api/uct.h

```diff
@@ -2626,6 +2630,40 @@ UCT_INLINE_API ucs_status_t uct_ep_am_short(uct_ep_h ep, uint8_t id, uint64_t he
 }
 
 
+/**
+ * @ingroup UCT_AM
+ * @brief Short send operation.
+ *
+ * This routine sends a message using @ref uct_short_protocol_desc "short" protocol.
+ * The input data in @a iov array of @ref ::uct_iov_t structures is sent to remote
+ * side ("gather output"). Buffers in @a iov are processed in array order.
+ * This means that the function completes iov[0] before proceeding to
+ * iov[1], and so on.
+ *
+ * @param [in] ep              Destination endpoint handle.
+ * @param [in] id              Active message id. Must be in range 0..UCT_AM_ID_MAX-1.
+ * @param [in] iov             Points to an array of @ref ::uct_iov_t structures.
+ *                             The @a iov pointer must be a valid address of an array
+ *                             of @ref ::uct_iov_t structures. A particular structure
+ *                             pointer must be a valid address. A NULL terminated
+ *                             array is not required. @a stride and @a count fields in
+ *                             @ref ::uct_iov_t structure are ignored.
+ * @param [in] iovcnt          Size of the @a iov data @ref ::uct_iov_t structures
+ *                             array. If @a iovcnt is zero, the data is considered empty.
+ *                             @a iovcnt is limited by @ref uct_iface_attr_cap_am_max_iov
+ *                             "uct_iface_attr::cap::am::max_iov".
+ *
+ * @return UCS_OK              Operation completed successfully.
+ * @return UCS_ERR_NO_RESOURCE Could not start the operation due to lack of
+ *                             send resources.
```

Comment:
@brminich @rakhmets wdyt about adding the following as we did for `uct_ep_tag_eager_bcopy()`?
```
* @return otherwise - Error code.
```

Reply:
I don't have any objection to that.

Reply:
@brminich wdyt?

Reply:
sounds good. For instance, it can return `UCS_ERR_INVALID_PARAM` if parameters check is enabled

Reply:
> sounds good. For instance, it can return `UCS_ERR_INVALID_PARAM` if parameters check is enabled

yep, or `UCS_ERR_IO_ERROR` from `ucs_socket_send_nb()`